[2025-02-23T01:07:48.456+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T01:07:48.489+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data manual__2025-02-23T01:07:45.535230+00:00 [queued]>
[2025-02-23T01:07:48.501+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data manual__2025-02-23T01:07:45.535230+00:00 [queued]>
[2025-02-23T01:07:48.501+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T01:07:48.518+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 01:07:45.535230+00:00
[2025-02-23T01:07:48.524+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=9564) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T01:07:48.526+0000] {standard_task_runner.py:72} INFO - Started process 9565 to run task
[2025-02-23T01:07:48.526+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'transform_weather_data', 'manual__2025-02-23T01:07:45.535230+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpmb9yvqf5']
[2025-02-23T01:07:48.527+0000] {standard_task_runner.py:105} INFO - Job 67: Subtask transform_weather_data
[2025-02-23T01:07:48.593+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.transform_weather_data manual__2025-02-23T01:07:45.535230+00:00 [running]> on host ee2081ae3a04
[2025-02-23T01:07:48.732+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T01:07:45.535230+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T01:07:45.535230+00:00'
[2025-02-23T01:07:48.733+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T01:07:48.735+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': 8.1, 'windspeed': 7.5, 'winddirection': 197, 'weathercode': 0}
[2025-02-23T01:07:48.775+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T01:07:48.775+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=transform_weather_data, run_id=manual__2025-02-23T01:07:45.535230+00:00, execution_date=20250223T010745, start_date=20250223T010748, end_date=20250223T010748
[2025-02-23T01:07:48.860+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T01:07:48.893+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T01:07:48.896+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
